### ISSUE TITLE
Topic filtering button for organizations

### ISSUE THREAD

**Issue by wouter-heerwegh** [refined-github/refined-github#5682]
### Description

I've been struggling to keep a clear overview of repositories in an organization I am part of. Github allows to add topics to repo's and you can filter on them by using `topic:topic_name` in the search bar in the organization. 

I would like to propose a feature that adds a button to show all the topics inside the organization and tp filter topics in the repository 

### Screenshot

![image](https://user-images.githubusercontent.com/20578313/172188702-0488adc1-34c7-45ee-8798-88947366db3d.png)

### Example URL

https://github.com/refined-github

**Comment by fregante** [comment]
They're kind of there already, even if they're not *all* visible 

https://github.com/microsoft

![image](https://user-images.githubusercontent.com/1402241/172078395-27da2747-e020-4d91-bbb9-bfb93e9eb31e.jpeg)

**Comment by wouter-heerwegh** [comment]
Would it be generic enough if this is also added to the repositories tab of a user? For instance: https://github.com/wouter-heerwegh?tab=repositories

For microsoft, it shows some of the topics on the main page, however, the topics are not shown on the repo's themselves. This is only visible on the repositories page, but there you cannot see an overview of all topics.

**Comment by wouter-heerwegh** [comment]
I made a very basic and unpolished version on a fork:
wouter-heerwegh/refined-github@36bc25e7c7da767d41256047045dedd06976af52
![image](https://user-images.githubusercontent.com/20578313/172188992-d993b406-a9f5-4a3b-89db-414644103a19.png)

Maybe it would also nice to see used topics in an organization/user profile when adding these to a repository

**Comment by Julian-Dumitrascu** [comment]
I want to choose a topic for a repository.
I've reported to GitHub that I can't do this because they present another interface in [their instructions](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics#adding-topics-to-your-repository).
I've tried to check (chrome://extensions/?options=hlepfoohegkhhmjieoechaddaejaokhf) whether the Refined GitHub is different. I can't say whether your extension is preventing me from seeing that About panel. Is it?
